README.md

```diff
@@ -116,18 +116,120 @@ One of the big concerns with IPFS, and even cloud data storage in general is enc
 
 ## Usage and Features
 
-### Spinning up a Node
+Before attempting to use Temporal you will need to install it. Even if you are going to be using our dockerized tooling, an install of Temporal is needed primarily for configuration file initialization.
```

Comment:
What is the roadmap to where this is not needed? It seems real silly - why do all the work to install just to get a config file to use in a docker, then uninstall the application :confused:

Reply:
should have an issue open on this IMHO :pray:

Reply:
maybe it should be in the `setup/scripts/misc/install_from_source.sh` script?

Reply:
Yes you're right, I think this can be resolved by way of a docker image entrypoint script that will check for a config file, and if one isn't present create one.

This does however mean it should generate sensible defaults so one doesn't have to manually edit the config file.

The entrypoint script will also need to handle appropriate migration of the postgresql tabels. I will create a github issue about this.